src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    public void draw();

```

Comment:
```suggestion
  void draw();
```
this method is public by default